documentation/deliverables/individual_deliverables/sprint_reporting_dashboard.md

```diff
@@ -0,0 +1,264 @@
+- **Why:** Publishing our metrics publicly encourages us to define and track key measures for each deliverable and builds public trust in the approach we’re taking. And tracking metrics around sprint and delivery, in particular, will help us scope and plan future deliverables more accurately by enabling us to understand team capacity and delivery velocity.
+- **Who:**
+  - Project maintainers who want to monitor team capacity and delivery velocity
+  - Internal and external stakeholders who want to monitor our progress toward key deliverables
+
+## Goals
+
+### Business description & value
+<!-- Required -->
+
+**Problem statement:** Establishing and actively tracking operational metrics is critical to the long-term success of software projects. Often projects will start strong, quickly delivering features that add value and improve the user experience of their products. Without the accountability and insight that operational metrics provide, though, delivery velocity and value can decrease over time due to a number of factors, such as technical debt and deferred maintenance. And without a way to meaningfully engage with these metrics, such as a dashboard, trends or signals in the data can easily go undetected for extended periods of time, making it harder to address the underlying cause.
+
+**Value:** Establishing robust operational metrics and publishing them to a public-facing dashboard are important steps in creating a data-driven culture that is focused on delivery. Defining clear operational metrics helps align the team internally around key delivery goals and creates a shared understanding around how we measure progress to those goals. Making these metrics publicly available commits us to tracking and monitoring our operational data and encourages us to respond to changes in performance against those metrics. By setting up the appropriate foundation to collect and publish data for delivery metrics, we also make it easier to publish future, more important, metrics about operational and program outcomes.
+
+**Goals**
+
+- Ensure that we are collecting the data needed to calculate proposed metrics
+- Begin to create the infrastructure required for more advanced data analytics
+- Begin to establish a framework to measure project impact and success
+- Build public trust in our approach through transparency around core metrics
+- Improve our ability to understand and manage team capacity and delivery velocity
+
+### User Stories
+<!-- Required -->
+
+- As a **project maintainer**, I want:
+  - to be able to provide input on which metrics we publish and how they are presented in the dashboard, so that I can use the dashboard to meaningfully plan and manage delivery on the project.
+  - to be able to provide additional details about the metrics included in the dashboard, so that other stakeholders know how to interpret these metrics correctly in the context of the project.
```

Comment:
Can we add something about being able to easily add new metrics in the future? 

Just trying to have something focused on developer joy and ease of updates.

Reply:
Good call! I added another user story in this commit: https://github.com/HHS/simpler-grants-gov/pull/516/commits/9652deea7dd042058dfea99b6c7a5be27738e49d